src/components/common/dataset-label.tsx

```diff
@@ -31,15 +32,12 @@ const DatasetName = styled.div.attrs({
 `;
 
 interface DatasetLabelType {
-  dataset: {
```

Comment:
Is this deletion correct?

Reply:
let't keep the original minimum type, so that `DatasetLabel` can be used as a generic component that does NOT require dataset to be a `KeplerTable`